### PR TITLE
Bugs fixing and run pipeline

### PR DESCRIPTION
**Pull request by YanetABUC**

### CODE REVIEW

File: src/plugins/reconness.js [hunk 1]
```diff
@@ -77,10 +77,11 @@ export default {
       }
     )
 
-    app.config.globalProperties.$agentStatus = Object.freeze(
+    app.config.globalProperties.$entityStatus = Object.freeze(
       {
         RUNNING: 1,
-        FINISHED: 2
+        FINISHED: 2,
+        WAITING: 3
```
Comment (gorums) [inline src/plugins/reconness.js:84]: We dont need WAITING

File: src/components/Pipelines/ExecutionAreaPipelineAgent.vue [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+      :class="{'p-0': true}"
+      style="position: relative;">
+      <div :class="{'invisible': Object.keys(item2).length === 0}">
+        <div class="info-box-background float-left w-65"
+          style="position: relative; left: 0px; top: -1px;">
+        </div>
+        <div :class="{ 'invisible': index+1 === associatedAgents.length || Object.keys(associatedAgents[index+1]).length === 0}"
+          class="mt-3 margin-center abs-center border-top w-35"
+          style="color:black!important;border: 1px solid; float:left"> </div>
+          <div v-if="index+1 !== associatedAgents.length" :class="{ 'invisible': index+1 === associatedAgents.length || Object.keys(associatedAgents[index+1]).length === 0 }" class="mt-3 black-circle">  </div>
```
Comment (gorums) [inline src/components/Pipelines/ExecutionAreaPipelineAgent.vue:13]: This is very hard to understand
```
"index+1 !== associatedAgents.length" 
:class="{ 'invisible': index+1 === associatedAgents.length || Object.keys(associatedAgents[index+1]).length === 0 }"
```
Can we move that to a method and call the method instead ??
And use better conversion name

File: src/components/Pipelines/ExecutionAreaPipelineAgent.vue [hunk 3]
```diff
@@ -0,0 +1,372 @@
...
+          <div>
+          <div  class="workflow-tools info-box invisible">
+            <div class="info-box-content">
+              <span data-toggle="modal" data-target="#confirmation-modal" :class="{'disabled' : (this.AgentsPipelineList).length === 1}" class="cursor-pointer material-icons" style="color: #ff4545 " @click="this.$store.commit('pipelines/changeValueToDelete', {idFather: item2.id, idSon: -1})"><a>cancel</a></span>
```
Comment (gorums) [inline src/components/Pipelines/ExecutionAreaPipelineAgent.vue:17]: instead item2 we can use better name like ```father```

File: src/components/Pipelines/ExecutionAreaPipelineAgent.vue [hunk 4]
```diff
@@ -0,0 +1,372 @@
...
+              <span data-toggle="modal" data-target="#pipelinesModalForm" @click="this.$store.commit('pipelines/changeIsBranchFather', -1)" style="color: #00B1FF" class="cursor-pointer material-icons">add_circle</span>
+              <span class="material-icons cursor-pointer" @click="onEdit" data-toggle="modal" data-target="#agentConfiguration" :data-id="item2.id">settings</span>
```
Comment (gorums) [inline src/components/Pipelines/ExecutionAreaPipelineAgent.vue:19]: same here

File: src/components/Pipelines/ExecutionAreaPipelineAgent.vue [hunk 5]
```diff
@@ -0,0 +1,372 @@
...
+            </div>
+          </div>
+          <button :id="'b' + index"  data-toggle="modal" data-target="#pipelinesModalForm"
+                      style="color: #00B1FF;" class="invisible  add-border btn create-agent-buttons-main-action"
+                      data-dismiss="modal"  @click="this.$store.commit('pipelines/changeIsBranchFather', index)">Add +
+                    </button>
+        </div>
+        <!-- <PipelineAgentMainFront @startRunningSons="secondaryProcessAgentsExecution" @pipelineAgentDone="processAgentsExecution" :totalItems="AgentsPipelineList.length" :index="index" :indexRunningAgent="indexRunningAgent" :startMainProcess="startMainProcess" :item2="item2"/> -->
+        <PipelineAgentMainFront :index="index" :item2="item2" :pipeline="pipeline"/>
```
Comment (gorums) [inline src/components/Pipelines/ExecutionAreaPipelineAgent.vue:28]: same here

File: src/components/Pipelines/ExecutionAreaPipelineAgent.vue [hunk 6]
```diff
@@ -0,0 +1,372 @@
...
+        <div>
+          <div>
+          <div class="pipeline-run-line" v-if="this.getAgentBranch(index).length !== 0"></div>
+            <div  v-for="(item3, index1) of this.getAgentBranch(index-1)"  :id="'branch' + index1" :key="item3.id" class= "agent-branch col-lg-12 col-xl-6 float-left p-0" style="position: relative;" >
```
Comment (gorums) [inline src/components/Pipelines/ExecutionAreaPipelineAgent.vue:32]: better names here

**Review by gorums — APPROVED** [review]

File: src/plugins/reconness.js (hunk 1 above)
Comment (YanetABUC) [reply src/plugins/reconness.js:84]: We decided to keep Waiting status to identified an agent that was executed and finished

**Pull request merged**